virtualization/windowscontainers/manage-containers/container-networking.md

```diff
@@ -14,32 +14,33 @@ ms.assetid: 538871ba-d02e-47d3-a3bf-25cda4a40965
 > ***DISCLAIMER: Please reference [Docker Container Networking](https://docs.docker.com/engine/userguide/networking/) for general docker networking commands, options, and syntax.*** With the exception of any cases described below, all Docker networking commands are supported on Windows with the same syntax as on Linux. Please note, however, that the Windows and Linux network stacks are different, and as such you will find that some Linux network commands (e.g. ifconfig) are not supported on Windows.
 > ## Unsupported features and network options 
 > The following networking options are currently **NOT** supported in Windows:
+>
 > | Command        | Unsupported Option   |
 > | ---------------|:--------------------:|
 > | ``docker run``|   ``--ip6``, ``--dns-option`` |
-> | ``docker network create``| ``--aux-address``, ``--internal``, ``--ip-range``, ``--ipam-driver``, ``--ipam-opt``, ``--ipv6``, ``--opt encrypted``      |
+> | ``docker network create``| ``--aux-address``, ``--internal``, ``--ip-range``, ``--ipam-driver``, ``--ipam-opt``, ``--ipv6``, ``--opt encrypted`` |
 
 
 ## Basic networking architecture
 This topic provides an overview of how Docker creates and manages networks on Windows. Windows containers function similarly to virtual machines in regards to networking. Each container has a virtual network adapter (vNIC) which is connected to a Hyper-V virtual switch (vSwitch). Windows supports five different networking drivers or modes which can be created through Docker: *nat*, *overlay*, *transparent*, *l2bridge*, and *l2tunnel*. Depending on your physical network infrastructure and single- vs multi-host networking requirements, you should choose the network driver which best suits your needs.
 
-<figure>
-  <img src="media/windowsnetworkstack-simple.png">
-</figure>  
+
+![text](media/windowsnetworkstack-simple.png)
+
 
 The first time the docker engine runs, it will create a default NAT network, 'nat', which uses an internal vSwitch and a Windows component named `WinNAT`. If there are any pre-existing external vSwitches on the host which were created through PowerShell or Hyper-V Manager, they will also be available to Docker using the *transparent* network driver and can be seen when you run the ``docker network ls`` command.  
 
-<figure>
-  <img src="media/docker-network-ls.png">
-</figure>
+
+![text](media/docker-network-ls.png)
+
 
 > - An ***internal*** vSwitch is one which is not directly connected to a network adapter on the container host 
 
 > - An ***external*** vSwitch is one which _is_ directly connected to a network adapter on the container host  
 
-<figure>
-  <img src="media/get-vmswitch.png">
-</figure>
+
+![text](media/get-vmswitch.png)
```

Comment:
We might want to include documenting usage and how to read Get-HNSNetwork and Get-HNSEndpoint since the VMSwitch cmdlets aren't always included unless Hyper-V management role is installed with management tools.